tools/clang/test/CodeGenHLSL/declarations/structs/declaration_in_return_type.hlsl

```diff
@@ -0,0 +1,13 @@
+// RUN: %dxc -E main -T vs_6_0 %s | FileCheck %s
+
+// Tests that struct declarations cannot also declare functions.
+// Note that FXC allows this
```

Comment:
It would be better to have a list of these incompatibilities in a text/markdown file in the doc folder with proper explanation on what is different and why. Not in test file names.

Reply:
well I was hoping that some searchable info in the test would make it easier when we want to compile the list in a GitHub doc.. anyway, I am fine if we just want to maintain it as a doc. Do we have a GitHub work item for that?

Reply:
How about a wiki page? Incompatibilities are not necessarily tied to code changes, we rather document them as we find them.

Reply:
sound good @MrTrillian. Could you please create a work-item (if it doesn't exit already) so that we don't forget?

Reply:
Better yet, I created the page. Feel free to add stuff you're aware of: https://github.com/Microsoft/DirectXShaderCompiler/wiki/Breaking-Changes-from-FXC

Reply:
Oh, we raced between our comments @vcsharma and I didn't see yours on the searchable info. Actually I like the idea to have a test folder `fxc_breaking_changes` with tests expecting errors but known to work with FXC. That's better than a wiki page. That folder should probably be in HLSL and have syntax-only tests.